test/dummy/lib/design_system/elements/nav_item/nav_item_facade.rb

```diff
@@ -0,0 +1,23 @@
+class NavItemFacade < FenrirView::Presenter
+  properties :title, :link, :style, :current_page
```

Comment:
Add another different path for `system_components` with it's own helper?

So they don't show up next to other components in the style guide.